package.json

```diff
@@ -65,7 +65,7 @@
         "@babel/preset-flow": "^7.14.5",
         "@trezor/blockchain-link": "^2.1.1",
         "@trezor/connect-common": "^0.0.4",
-        "@trezor/rollout": "^1.3.1",
+        "@trezor/rollout": "~1.2.0 || ^1.3.2",
```

Comment:
this change should not be needed anymore I think. latest rollout should contain your fix allowing it to run with node 12

Reply:
Done, I've deleted that commit, after rebasing to `develop`.